### PR TITLE
Include 'create_potransaction' in the list of available functions

### PR DESCRIPTION
**Pull request by sam-peach**
Include 'create_potransaction' in the list of available functions

### CODE REVIEW

File: lib/intacct_ruby/function.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@ class Function
       update
       delete
       getAPISession
+      create_potransaction
```
Comment (hoodr) [inline lib/intacct_ruby/function.rb:17]: totally NB: I don't want to be that guy, but why not `po_transaction`?

```suggestion
      create_po_transaction
```

**Review by hoodr — APPROVED** [review]

File: lib/intacct_ruby/function.rb (hunk 1 above)
Comment (sam-peach) [reply lib/intacct_ruby/function.rb:17]: @hoodr it's to match the name of the function that the Sage API then calls
```
<create_potransaction>
    <transactiontype>Purchase Requisition</transactiontype>
    <datecreated>
...
```

Comment (tmferretti) [reply lib/intacct_ruby/function.rb:17]: @sam-peach is right, if you look at how Function.rb works, youll see it basically gets turned into an xml tag

**Review by tmferretti — APPROVED** [review]

**Comment by tmferretti** [comment]
watched sam tophat this a few days ago

**Pull request merged**